### PR TITLE
Integrate litespi modules

### PR DESCRIPTION
**Pull request by kowalewskijan**
These changes introduce integration of generated modules with generic layer of LiteSPI.

@enjoy-digital @mithro please take a look.

### CODE REVIEW

**Comment by kowalewskijan** [comment]
This should be merged after this [PR](https://github.com/litex-hub/litespi/pull/1)

File: litespi/phy/generic.py [hunk 1]
```diff
@@ -78,12 +73,30 @@ def shift_out(self, width, bits, next_state, trigger=None, op=None, ddr=False):
         return res
 
     def __init__(self, pads, flash=None, device="xc7"):
+        assert flash is not None
```
Comment (piotr-binkowski) [inline litespi/phy/generic.py:76]: I think we can remove the `assert` statement and remove `flash` default value to get the same result

**Comment by kowalewskijan** [comment]
I fixed the "assert" issue. @piotr-binkowski please merge if rest of the code is ok.

**Comment by piotr-binkowski** [comment]
Looks fine to me, I don't have the permissions so @enjoy-digital or @mithro will need to take a look and merge this

**Comment by enjoy-digital** [comment]
@kowalewskijan, @piotr-binkowski: you should now have the permissions to merge PRs. For large PRs where you think it's useful to have a review, you can ask us to review and merge. For simple fixes or commits, feel free to commit/merge directly.

**Pull request merged**